docs/module.rst

```diff
@@ -15,19 +15,25 @@ name of the file, but also the position of the file under the *library
 root* (a folder under which the standard library and all packages are
 installed).
 
+By default, the library root is ``/usr/local/lib/lambdapi/lib_root`` or
```

Comment:
Is it a valid path under all Linux systems? What about Windows or Mac?

Reply:
It must be valid under most Linux system (since it's mentioned in https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) and it's valid for BSD systems (https://en.wikipedia.org/wiki/Unix_filesystem#Conventional_directory_layout) and hence MacOS. I don't know for windows...